docs/content/preview/faq/yugabytedb-managed-faq.md

```diff
@@ -138,7 +138,7 @@ To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte
 
 Yugabyte only upgrades clusters during scheduled maintenance windows. Yugabyte notifies you in advance of any upcoming upgrade via email. The email includes the date and time of the maintenance window. An Upcoming Maintenance badge is also displayed on the cluster. You can start the upgrade any time by signing in to YugabyteDB Managed, selecting the cluster, clicking the **Upcoming Maintenance** badge, and clicking **Upgrade Now**. To delay the maintenance, click **Delay to next available window**. To manage maintenance windows, select the cluster [Maintenance tab](../../yugabyte-cloud/cloud-clusters/cloud-maintenance/).
 
-The database is upgraded to the latest release in the [release track](#what-version-of-yugabytedb-does-my-cluster-run-on) that was selected when the cluster was created (either preview or stable). Sandbox clusters are always in the preview track.
+The database is upgraded to the latest release in the [release track](#what-version-of-yugabytedb-does-my-cluster-run-on) that was selected when the cluster was created (either Innovation or Production, or Preview for Sandbox clusters). Sandbox clusters are always in the preview track.
```

Comment:
Can we rephrase this to avoid confusion? I missed the comma and read this as: Innovation or Production or Preview for Sandbox clusters.